text/0032-nft-standard.md

```diff
@@ -117,7 +199,30 @@ The AttributeObject may have more attributes than the ones described above but t
             "type": "ring",
             "value": 0
         }
-    ]
+    ],
+   "royalty": {
+        "creator_royalty_basis_points": 500,
+        "creators": [
+              {
+                "address": "H7t2eNhFNeH3hrUAJj4AAu9amtWmPT4fLB",
+                "share": 95
+              },
+              {
+                "address": "HNBuYEsJYznh5zZGx4eDn2NXYuiPpgKBgH",
+                "share": 5
+              },
+              {
+                "address": "RaNd0MsJYznh5zYOyoDn2NXYuiPAdDre5s",
+                "share": 0
```

Comment:
In this case, the idea is just to identify this address as a creator, even if it's not receiving any royalties?